.jenkinsci/text-variables.groovy

```diff
@@ -200,6 +200,17 @@ cmd_description = """
          </li>
       </ul>
    </li>
+   <li>
+      <p><strong>build_shared_libs</strong> = false&nbsp;</p>
```

Comment:
This options require to set additional options `testList = '()'`, that mean we should color it to red.
do it also conflict with option `x64linux_compiler_list= ['clang6']`?
```suggestion
      <p><span style="color: #ff0000;"><strong>build_shared_libs</strong></span> = false&nbsp;</p>
```

Reply:
It should work fine with all Linux and Mac OS compilers, but it is enough to use for one compiler only at the moment.